scripts/correct_video_timestamps.py

```
@@ -0,0 +1,271 @@
+        tags = data['format']['tags']
+        # Check common timestamp fields
+        for key in ['creation_time', 'date', 'DATE']:
+            if key in tags:
+                creation_time = tags[key]
+                break
+    
+    if creation_time:
+        # Parse ISO format
+        try:
+            return datetime.fromisoformat(creation_time.replace('Z', '+00:00'))
+        except:
```

Ruff (0.15.14)

[error] Do not use bare `except`

(E722)